SPECS/xs-opam-repo.spec

```diff
@@ -29,6 +29,9 @@ Source0: xs-opam-repo-6.74.0.tar.gz
 # when you want a new build.
 # Source1: https://code.citrite.net/rest/archive/latest/projects/~YOURUSER/repos/ezxenstore/archive?at=master&format=tar.gz&prefix=ezxenstore#/ezxenstore.tar.gz
 
+# XCP-ng backport xapi-project/xen-api#5456
+Patch0: metajson.patch
```

Comment:
We have a naming convention for the patches, this should be: `xs-opam-repo-6.74.0-metajson.XCP-ng.patch`.

Reply:
done

Reply:
I was writing a comment. Actually a better name would indicate what the patch changes.

Maybe: 
`xs-opam-repo-6.74.0-patch-metajson-to-fix-xrrd-stats.XCP-ng.patch`

Or, since it's a backport from upstream:

`xs-opam-repo-6.74.0-patch-metajson-to-fix-xrrd-stats.backport.patch`